db/models.py

```diff
@@ -50,3 +54,69 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE,
+                             related_name="orders"
+                             )
+
+    class Meta:
+        ordering = ["-created_at"]
```

Comment:
Code Style Issue: The `class Meta` is implemented correctly, but it's best to define it after all the fields of the model. This is just a code style suggestion and won't affect the functionality of your code.